src/pages/Customer/Cart/CartPage.js

```diff
@@ -0,0 +1,204 @@
+import React, { useState } from "react";
+import { Minus, Plus, X } from "react-feather";
+
+import InputSelect from "components/InputControl/InputSelect/InputSelect";
+
+import styles from "./CartPage.module.scss";
+
+const CartPage = () => {
+  const [quantity, setQuantity] = useState(1);
+
+  return (
+    <div className={styles.outer}>
+      <div className={styles.leftPortion}>
+        <div className={styles.shopTitle}>
+          Your Shopping Cart <span>(2 Items)</span>
+        </div>
+        <div className={styles.leftInner}>
+          <div className={styles.productDetailsbox}>
+            <div className={styles.productsList}>
+              <div className={styles.productCard}>
+                <div className={styles.productInfo}>
+                  <div className={styles.productImgbox}>
+                    <img
+                      src="https://images.pexels.com/photos/672101/pexels-photo-672101.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500"
+                      alt="productName"
+                      className={styles.productImg}
+                    />
+                    <div className={styles.discountPrice}>-5%</div>
+                  </div>
+                  <div className={styles.aboutBox}>
+                    <div className={styles.productName}>
+                      Apples Lorem ipsum dolor sit amet.
+                    </div>
+                    <div className={styles.availableIn}>5kg</div>
+                    <div className={styles.productPrice}>₹ 30.9</div>
+                  </div>
```

Comment:
line no 19-162 ye hard code nhi karte

isko map karke kiya ja sakta hai vo acha tarika hai

like make a array of products=[{...},{...},{...}]
then map through this array to print all these.

Reply:
to design to bnaya h api to iske baad hito add hogi
tbhi to data aayega